rfcs/text/0000-Managed-Access-Data.md

```diff
@@ -0,0 +1,346 @@
+results for which I have permission.
+
+As a Researcher with a pipette, I want to use the Data Browser and/or
+tertiary analysis portals to look for interesting patterns of a gene or
+cell type expression. For example, I might look for all the tissues
+where my gene of interest is expressed, or try to find a combination of
+gene expressions to identify a pathway in my disease of interest. I
+would like this to work while enforcing the access permissions for the
+data, returning results for which I have permission.
+
+As a Researcher with a pipette, I would like to be able to share all my
+cellular resolution, human genomic data with the HCA DCP, including data
+which needs to be distributed via managed access as well as data which
+can be openly released. I want this to support the aim to build an atlas
+of all human cells.
+
+As a researcher with a pipette, I would like to manage a Data Access
+Committee (DAC) so that I can control access to individuals and groups
+as follows: (NOTE: this is the researcher-run DAC model, which is the
+least burden to the DCP, but the highest burden to the researcher)
+
+1.  (Submitter/Wrangler role) I would like to grant or revoke read-write
+    > access to individuals or groups for data submission purposes
+
+2.  (Consumer role) I would like to grant or revoke read-only access to
+    > individuals or groups based on the type of use they are requesting
+
+3.  (Admin role) I would like to grant or revoke read-write access to
+    > individuals or groups for data management purposes (including
+    > possible redaction of data)
+
+The objects being controlled are files, bundles, and collections, as
+well as setting permissions for all objects related to a project.
+
+As a researcher with a pipette, I would like to set an embargo for
+access to my data so that I may finish my research and publish results
+based on my data before others. I would like to restrict access to my
+data for a set period of time.
+
+**Scientific \"guardrails\"** 
+------------------------------
+
+This plan will be reviewed by members of the [Compliance working
+group]([[https://github.com/HumanCellAtlas/dcp-community/blob/master/charters/Compliance-WG/charter.md]](https://github.com/HumanCellAtlas/dcp-community/blob/master/charters/Compliance-WG/charter.md))
+who will ensure the proposed design meets their guidelines for
+responsible handling of managed access genomic data.
+
+**Detailed Design**
+-------------------
+
+A tiered access model:
+
+**Open Access**: The HCA project heavily favors open-access data. This
+is the primary use case for the DCP. Users will not be required to log
+in to gain access to any openly consented data. Searches executed for
+unauthenticated users will only return open access data.
+
+**Controlled Access**: To access controlled data the user will have to
+log in to the system to identify themselves. Authentication will happen
+via OIDC mechanisms. Once authenticated, the user's searches will return
```

Comment:
I do not see a definition for OIDC - so at the very least, I would reference this as:

_OpenID Connect (OIDC)_ and perhaps even link to the appropriate documentation.  Generally, use the full name in the first instance of associating with its acronym for further use.

Reply:
thanks